native/common/include/jp_primitive_accessor.h

```diff
@@ -206,6 +206,11 @@ class JPConversionLongWiden : public JPConversion
 {
 public:
 
+	virtual JPMatch::Type matches(JPMatch &match, JPClass *cls) override
+	{
+		return JPMatch::_none;  // GCOVR_EXCL_LINE not used
+	}
+
```

Comment:
But we end up paying for the pure virtual by requiring two extra stub methods here and in the next template because these require the type dependent switch statement and thus sadly cant become match methods.